src/Console/Command/FixCommand.php

```diff
@@ -169,6 +169,9 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
             $configFile = $resolver->getConfigFile();
             $stdErr->writeln(sprintf('Loaded config <comment>%s</comment>%s.', $resolver->getConfig()->getName(), null === $configFile ? '' : ' from "'.$configFile.'"'));
+            if (OutputInterface::VERBOSITY_VERBOSE <= $output->getVerbosity() && \PHP_SAPI !== 'phpdbg' && !\defined('HHVM_VERSION')) {
```

Comment:
out of curiosity. why checking phpdbg/hhvm ?

Reply:
The line will say something like "Runtime: PHP 7.4.10" - with phpdbg/hhvm the runtime would not be PHP, but phpdbg/hhvm, right?

Instead of calculating what is the runtime I went with the assumption that not many people would ever usu PHP CS Fixer on phpdbg/hhvm, so we can ignore these cases.

// cc @SpacePossum

Reply:
If people run the tool on HHVM they are on their own if you ask me  :)  I remember a lot of effort of people here trying to support HHVM but in the end "they" (HHVM developers) were not interested and we had to drop support.

For `phpdbg` I think we are OK with showing the value after all. Even if people are running on exotic builds of PHP or with all kinds of extensions; having that as information displayed here would be of value for us.

Reply:
Make sense.

BTW Do you feel like approving this PR? Right now you both requested changes and added RTM label 😄

Reply:
>requested changes and added RTM label

sounds like me already :D